client/my-sites/plugins/plugins-browser-list/test/index.jsx

```diff
@@ -20,6 +24,7 @@ const plugins = [
 
 const props = {
 	plugins,
+	variant: PluginsBrowserListVariant.Fixed,
```

Comment:
It doesn't look like we're passing in the other variants as props, why is it needed here?

Reply:
The prop `variant` is marked as required in `PluginsBrowserList` and should be one of the values [listed here](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plugins/plugins-browser-list/types.ts)